includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -180,72 +180,6 @@ public function __construct( $dom, $args = [] ) {
 
 		parent::__construct( $dom, $args );
 
-		// @todo AMP dev mode should eventually be used instead of allow_dirty_styles.
-		if ( ! empty( $this->args['allow_dirty_styles'] ) ) {
```

Comment:
This code should be removed too:

https://github.com/ampproject/amp-wp/blob/a8a4d4d05f9727aad2304db103a3ab97817aafa6/includes/sanitizers/class-amp-style-sanitizer.php#L847-L851

Any reference to `allow_dirty_styles` and `allow_dirty_scripts` can be removed.

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/4977/commits/8084a3f5ab392ba717780dc8687805f8872204e5.